lib/Dialect/HW/HWOps.cpp

```diff
@@ -1710,6 +1710,13 @@ LogicalResult ArrayCreateOp::verify() {
   return success();
 }
 
+Optional<Value> ArrayCreateOp::getUniformElement() {
```

Comment:
Values are nullable, would it be more idiomatic to just return a `Value` instead of an `Optional<Value>`?

Reply:
Oh I'm such a derp. Totally!

Reply:
Done